lib/src/main/java/com/basistheory/android/view/SensitiveEditText.kt

```diff
@@ -22,8 +22,19 @@ internal class SensitiveEditText(
     }
 
     override fun getText(): Editable? {
-        return if (allowTextAccess || BuildConfig.DEBUG)
-            super.getText()
-        else SpannableStringBuilder()
+        val stackTrace = Thread.currentThread().stackTrace
+        val indexOfBtEditText =
+            stackTrace.indexOfLast { e -> e.className == "com.basistheory.android.view.SensitiveEditText" }
+        val isInternalInvocation =
+            stackTrace[indexOfBtEditText + 1].className.startsWith("android.widget.") || stackTrace[indexOfBtEditText + 1].className.startsWith(
```

Comment:
Is this index guaranteed to exist? This should probably be changed to get the item at that index safely since this is critical path and could cause app crashes if the stacktrace is unexpected

Reply:
from what I saw it should always be there but I added a try catch